recipes/aeron/all/conanfile.py

```diff
@@ -0,0 +1,161 @@
+import os
+import shutil
+import re
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class AeronConan(ConanFile):
+    name = "aeron"
+    description = "Efficient reliable UDP unicast, UDP multicast, and IPC message transport"
+    topics = ("conan", "aeron", "udp", "messaging", "low-latency")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/real-logic/aeron/wiki"
+    license = "Apache License 2.0"
+    exports_sources = "CMakeLists.txt",
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_aeron_driver": [True, False],
+        "build_tests": [True, False],
+        "build_samples": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "build_aeron_driver": False,
+        "build_tests": False,
+        "build_samples": False
+    }
+    generators = "cmake",
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+        if self.settings.os == "Windows":
+            self.requires("pthreads4w/3.0.0")
+            if self.options.shared:
+                raise ConanInvalidConfiguration("{} option shared=True is not supported on Windows", self.name)
+
```

Comment:
```suggestion

    def validate(self):
```

Validate should be used for minimal cpp standard.